src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -0,0 +1,125 @@
+    String defaultUserId = APITestContext.getUserId();
+
+    APITestContext.setUserId("null");
+    loansFixture.checkOutByBarcode(item, user, new DateTime(DateTimeZone.UTC));
+
+    loansFixture.checkInByBarcode(item, checkInDate, servicePointId);
+    JsonObject lastCheckIn = itemsClient.get(item.getId()).getJson().getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(nullValue()));
+
+    APITestContext.setUserId("");
+    loansFixture.checkInByBarcode(item, checkInDate, servicePointId);
+    lastCheckIn = itemsClient.get(item.getId()).getJson().getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(nullValue()));
+
+    APITestContext.setUserId(defaultUserId);
+  }
+
+
+  @Test
+  public void shouldBeAbleToCheckinItemWithoutLoan() throws InterruptedException,
+    MalformedURLException, TimeoutException, ExecutionException {
+
+    IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    DateTime checkInDate = DateTime.now();
+
+    loansFixture.checkInByBarcode(item, checkInDate, servicePointId);
+    JsonObject actualItem = itemsClient.get(item.getId()).getJson();
+
+    JsonObject lastCheckIn = actualItem.getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(APITestContext.getUserId()));
+  }
+
+  @Test
+  public void shouldBeAbleCheckinItemWithoutLoanMultipleTimes() throws InterruptedException,
+    MalformedURLException, TimeoutException, ExecutionException {
+
+    IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    DateTime firstCheckInDate = DateTime.now();
+
+    loansFixture.checkInByBarcode(item, firstCheckInDate, servicePointId);
+    JsonObject actualItem = itemsClient.get(item.getId()).getJson();
+
+    JsonObject lastCheckIn = actualItem.getJsonObject("lastCheckIn");
+
+    assertThat(lastCheckIn.getString("dateTime"), is(firstCheckInDate.toString()));
+    assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId.toString()));
+    assertThat(lastCheckIn.getString("staffMemberId"), is(APITestContext.getUserId()));
+
+    loansFixture.checkInByBarcode(item, DateTime.now(), servicePointId);
+    loansFixture.checkInByBarcode(item, DateTime.now(), servicePointId);
+    loansFixture.checkInByBarcode(item, DateTime.now(), servicePointId);
```

Comment:
I think this example could be improved by using a different service point in the later / last check in

Reply:
Done.